Week02/lesson03/homework/relaxr_starter_code/index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+
+	<meta charset="utf-8">
+	<meta name="description" content="Your description goes here">
+	<meta name="keywords" content="one, two, three">
+	<title>Relaxr Home</title>
+
+	<!-- external CSS links -->
+	<link rel="stylesheet" href="css/reset.css">
+	<link rel="stylesheet" href="css/style.css">
+  <link href='https://fonts.googleapis.com/css?family=Open+Sans:300,300italic,600,800' rel='stylesheet' type='text/css'>
+</head>
+
+<body>
+
+	<div id="container">
+		<header>
+
+			<div class="headerinner">
+				<h1 class="logo">Relaxr</h1>
+				<div class="cta">
+					<h2>Get piece of mind with a single tap</h2>
+					<a href="#" class="button">Get it Now</a>
+				</div>
+			</div>
+
+
+
+		</header>
+
+		<main>
+			<div class="benefits">
+				<div class="benefitsinner">
```

Comment:
`benefitsinner` and `headerinner` both have similar CSS:

```css
width: 75%;
margin-left: auto;
margin-right: auto;
```

... maybe `.container` would've been good instead?